extensions/kubectl-cli/src/extension.ts

```diff
@@ -327,7 +328,7 @@ async function postActivate(
   // check if there is a new version to be installed and register the updater
   const lastReleaseMetadata = await kubectlDownload.getLatestVersionAsset();
   const lastReleaseVersion = lastReleaseMetadata.tag.slice(1);
-  if (lastReleaseVersion !== version) {
+  if (neq(lastReleaseVersion, version)) {
```

Comment:
maybe we should uses `gt`, because with the current code if lastReleaseVersion is 1.15.0 and version is 1.99, we will register a previous version.